encrypted-payloads/README.md

```diff
@@ -0,0 +1,10 @@
+### Steps to run this sample:
+1) You need a Temporal service running. See details in README.md
+2) Run the following command to start the worker
+```
+go run worker/main.go
+```
+3) Run the following command to start the example
+```
+go run starter/main.go
+```
```

Comment:
4) Run
```
tctl workflow show --wid encrypted-payloads_workflowID
```
and observer encrypted agrs in the history.

Reply:
Please, just run it yourself first to make sure it works.

Reply:
This will show:
```
...
  11  WorkflowExecutionCompleted  {Result:[encoding
                                  binary/crypt: payload
                                  encoding is not supported],
                                  WorkflowTaskCompletedEventId:10}
```

So it won't actually show the raw bytes here, but I think that's ok?

Reply:
An alternative would be to mark this as a byteslice payload and use `metadata["wrapped-encoding"]` to detect if we should decrypt. That way `tctl` would show the bytes. What do you think?

Reply:
Switched to marking the encrypted payload as `binary/plain` so that the tools display the bytes.